libraries/AP_HAL_ChibiOS/hwdef/SPRacingH7/hwdef.dat

```diff
@@ -11,6 +11,8 @@ APJ_BOARD_ID 1060
 # crystal frequency, setup to use external oscillator
 OSCILLATOR_HZ 8000000
 
+FAST_MCU 1
```

Comment:
nice feature, but I think the name should be MCU_CLOCKRATE_MHZ with the value, for example:
```MCU_CLOCKRATE_MHZ 480```

Reply:
Fixed